stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -242,6 +237,23 @@ public final class _ExecutionContext {
     guard let opts = TFE_NewContextOptions() else {
       fatalError("ContextOptions object can never be nil.")
     }
+
+    // Create TF config object.
+    if _RuntimeConfig.executionMode == .xla {
+      debugLog("Enable XLA execution.")
+    }
+    if _RuntimeConfig.gpuMemoryAllowGrowth {
+      debugLog("Allowing growth for GPU memory allocator.")
+    }
+    self.tensorFlowConfig = TF_CreateConfig(
+      _RuntimeConfig.executionMode == .xla ? 1 : 0,
+      _RuntimeConfig.gpuMemoryAllowGrowth ? 1 : 0)
+    TFE_ContextOptionsSetConfig(opts,
+                                self.tensorFlowConfig.pointee.data,
```

Comment:
No need for self here and for other places.

Reply:
Done